pkgs/stdenv/adapters.nix

```diff
@@ -44,6 +44,7 @@ rec {
       then throw "Cannot build fully static binaries on Darwin/macOS"
       else stdenv'.mkDerivation (args // {
         NIX_CFLAGS_LINK = toString (args.NIX_CFLAGS_LINK or "") + " -static";
+        separateDebugInfo = false;
```

Comment:
This doesn't seem right, why must this be disabled for static adapter?

Reply:
Yeah maybe not... I was getting an issue that I thought was due to this but it might have been unrelated.